code-scanning/clj-holmes.yml

```
@@ -0,0 +1,40 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: clj-holmes
+
+on:
+  push:
+    branches: [ $default-branch, $protected-branches ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ $default-branch ]
+  schedule:
+    - cron: $cron-weekly
+
```

```suggestion
permissions:
  contents: read

```
We've started adopting the convention of providing a default minimal permission, as a defensive step against later adding addition jobs _without_ specifying permissions.